eo-maven-plugin/src/main/java/org/eolang/maven/name/OnVersioned.java

```diff
@@ -42,15 +43,34 @@ public final class OnVersioned implements ObjectName {
     /**
      * Default hash.
      */
-    private final CommitHash hsh;
+    private final Unchecked<CommitHash> hsh;
+
+    /**
+     * Put default hash forcibly.
+     */
+    private final boolean force;
```

Comment:
@maxonfjvipon Looks like a configurable object:
https://www.yegor256.com/2016/04/19/object-must-not-be-configurable.html